docs/EXTENDED-ARCHIVES.md

```diff
@@ -0,0 +1,428 @@
+etc.
+
+##### `colorScheme (Object)`
+
+The publisher specific color scheme, if any.
+
+The client may allow a certain degree of branding based on this color scheme.
+
+You may declare the following colors, in hexadecimal format:
+
+- `background`: The background color
+- `text`: The text color
+- `primary`: The primary color
+
+#### Device
+
+Here's an example of a real-world device manifest for Raspberry Pi 3:
+
+```json
+{
+  "displayName": "Raspberry Pi 3",
+  "url": "https://www.raspberrypi.org/products/raspberry-pi-3-model-b/",
+  "arch": "armv7hf",
+  "slug": "raspberrypi3",
+  "slugAliases": [
+    "raspberry-pi3",
+    "rpi3"
+  ]
+}
+```
+
+Describe a device with any of the following properties:
+
+##### `displayName (String)`
+
+The human-friendly name of the device.
+
+##### `url (String)`
+
+The url to the landing page of the device.
+
+##### `arch (String[])`
+
+The device architecture.
+
+The current possible values are: `aarch64`, `amd64`, `armv5e`, `armv6`,
+`armv7hf`, and `i386`.
+
+##### `slug (String)`
+
+The device slug. Officially recognised slugs include:
+
+- `artik10`
+- `artik5`
+- `artik710`
+- `beaglebone-black`
+- `beaglebone-green-wifi`
+- `beaglebone-green`
+- `colibri-imx6dl`
+- `cybertan-ze250`
+- `edge`
+- `hummingboard`
+- `intel-edison`
+- `intel-nuc`
+- `nitrogen6x`
+- `odroid-c1`
+- `odroid-xu4`
+- `parallella`
+- `qemux86-64`
+- `qemux86`
+- `raspberry-pi`
+- `raspberry-pi2`
+- `raspberrypi3`
+- `ts4900`
+- `ts7700`
+- `via-vab820-quad`
+- `zynq-xz702`
+
+##### `slugAliases (String[])`
+
+The device slug aliases, if any.
+
+##### `compatibleWith (String[])`
+
+An array of the device slugs this device is compatible with.
+
+#### Image
+
+Here's an example of a real-world image manifest for Raspbian Jessie:
+
+```json
+{
+  "displayName": "Raspbian Jessie",
+  "version": "May 2016",
+  "url": "https://www.raspberrypi.org/downloads/raspbian/",
+  "releaseNotesUrl": "http://downloads.raspberrypi.org/raspbian/release_notes.txt",
+  "checksumType": "sha1",
+  "recommendedDriveSize": 4294967296,
+  "updateUrl": "https://downloads.raspberrypi.org/raspbian_latest",
+  "etag": "c0170-53152af2-533d18ef29fc0",
+  "supportedDevices": [
+    "raspberrypi",
+    "raspberrypi2",
+    "raspberrypi3"
+  ]
+}
+```
+
+Describe an image with any of the following properties:
+
+##### `displayName (String)`
+
+The human-friendly name of the image.
+
+##### `version (String)`
+
+The version of the image.
+
+##### `url (String)`
+
+The main url of the image.
+
+##### `releaseNotesUrl (String)`
+
+The url to the image's release notes or CHANGELOG. This URL will take
+precedence over `.meta/release-notes.txt` if defined.
+
+##### `releaseDate (String)`
+
+The release date timestamp. The date should conform to [ISO 8601][iso8601]
+standard.
+
+##### `checksumType (String)`
+
+The checksum type. The current possible values are: `sha1`, `crc32`, and `md5`.
+
+See the `.meta/checksum` file.
+
+##### `recommendedDriveSize (Number)`
```

Comment:
Since we're going to impose ordering of the files in the archive, the actual file information will be accessible from the tar header right before the image (after the metadata), so I don't think also having it here has an advantage for us. Of course it'd be faster, but then is also a point where data can get out of sync, and we should try to prevent that from happening where possible.